documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,255 @@
+----
 
-[discrete]
-== Example YAML showing KafkaConnector configuration
+Setting `use-connector-resources` to `true`  enables KafkaConnectors to create, delete and reconfigure connectors.
+
+If enabled in your `KafkaConnect` configuration, you must use the `KafkaConnector` resource to define connectors.
+`KafkaConnector` resources are configured to connect to external systems.
+They are deployed to the same Kubernetes cluster as the Kafka Connect cluster and Kafka cluster interacting with the external data system.
+
+.Kafka components are contained in the same Kubernetes cluster
+image:overview/kafka-concepts-kafka-connector.png[Kafka and Kafka Connect contained in the same cluster]
+
+The configuration specifies how connector instances connect to an external system, including any authentication.
+You also need to say what data to watch within the data source.
+For example, you might provide a database name in the configuration.
+You can also specify where the data should sit in Kafka by specifying a target topic name.
+
+.Example KafkaConnector configuration
 [source,yaml,subs="attributes+"]
 ----
-apiVersion: {KafkaConnectorApiVersion}
+apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnector
 metadata:
-  name: my-source-connector
+  name: my-source-connector  <1>
   labels:
-    strimzi.io/cluster: my-connect-cluster
+    strimzi.io/cluster: my-connect-cluster <2>
 spec:
-  class: org.apache.kafka.connect.file.FileStreamSourceConnector
-  tasksMax: 2
-  config:
-    file: "/opt/kafka/LICENSE"
-    topic: my-topic
+  class: org.apache.kafka.connect.file.FileStreamSourceConnector <3>
+  tasksMax: 2 <4>
+  config: <5>
+    file: "/opt/kafka/LICENSE" <6>
+    topic: my-topic <7>
     # ...
 ----
+<1> Name of the `KafkaConnector` resource, which is used as the name of the connector. Use any name that is valid for a Kubernetes resource.
+<2> Name of the Kafka Connect cluster to create the connector instance in. Connectors must be deployed to the same namespace as the Kafka Connect cluster they link to.
+<3> Full name or alias of the connector class. This should be present in the image being used by the Kafka Connect cluster.
+<4> Maximum number of Kafka Connect tasks that the connector can create.
+<5> link:{BookURLDeploying}#kafkaconnector-configs[Connector configuration^] as key-value pairs.
+<6> Location of the connector configuration file. The source connector reads from the `/opt/kafka/LICENSE` file.
+<7> Kafka topic to publish the source data to.
 
-You enable KafkaConnectors by adding an annotation to the `KafkaConnect` resource.
-KafkaConnector resources must be deployed to the same namespace as the Kafka Connect cluster they link to.
+NOTE: You can link:{BookURLUsing}#proc-loading-config-with-provider-str[load confidential configuration values for a connector^] from Kubernetes Secrets or ConfigMaps.
 
 [discrete]
-== Example YAML showing annotation to enable KafkaConnector
-[source,yaml,subs="attributes+"]
+== Kafka Connect API
```

Comment:
Could we add some note here that if users want to access the API from outside Kubernetes they should create their own Service / Ingress / Route? That is a common question.

Reply:
We've got that now on line 249 below. But perhaps a small "Exposing the Kafka Connect REST API" procedure would be better (certainly more easily found).